src/lib.rs

```diff
@@ -581,6 +591,7 @@ impl Default for BindgenOptions {
             emit_ast: false,
             emit_ir: false,
             derive_debug: true,
+            derive_default: true,
```

Comment:
I have added a new option `with-derive-default` and disable it by default, when we use it on tests, just enable the hidden `no-derive-default` option which has higher priority.
```rust
    let prepend = ["bindgen",
                   "--with-derive-default",
                   header_str,
                   "--raw-line",
                   "",
                   "--raw-line",
                   "#![allow(non_snake_case)]",
                   "--raw-line",
                   ""];

    let args = prepend.into_iter()
        .map(ToString::to_string)
        .chain(flags.into_iter());

    builder_from_flags(args)
        .map(|(builder, _)| Some(builder.no_unstable_rust()))
```